Changelog.md

```diff
@@ -1,5 +1,28 @@
 ## Ignition Rendering
 
+### Ignition Rendering 6.6.1 (2023-09-01)
+
+1. Fixed light visual in OGRE
+    * [Pull request #864](https://github.com/gazebosim/gz-rendering/pull/#864)
```

Comment:
```suggestion
    * [Pull request #864](https://github.com/gazebosim/gz-rendering/pull/864)
```

Reply:
Done in https://github.com/gazebosim/gz-rendering/pull/891/commits/4747722e17a13b48977e4d9835af3fd49d461689